fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -18,46 +18,67 @@
 package org.apache.doris.common;
 
 public class Config extends ConfigBase {
-
-    /*
-     * This specifies FE log dir. FE will produces 2 log files:
-     * fe.log:      all logs of FE process.
-     * fe.warn.log  all WARNING and ERROR log of FE process.
-     */
-    @ConfField public static String sys_log_dir = System.getenv("DORIS_HOME") + "/log";
-    @ConfField public static String sys_log_level = "INFO"; // INFO, WARNING, ERROR, FATAL
-    /*
-     * The roll mode of FE log files.
-     * TIME-DAY:    roll every day.
-     * TIME-HOUR:   roll every hour.
-     * SIZE-MB-nnn: roll by size.
-     */
-    @ConfField public static String sys_log_roll_mode = "SIZE-MB-1024"; // TIME-DAY， TIME-HOUR， SIZE-MB-nnn
+    
     /*
-     * Maximal FE log files to be kept.
-     * Doesn't work if roll mode is TIME-*
+     * The max size of one sys log and audit log
      */
-    @ConfField public static int sys_log_roll_num = 10;
+    @ConfField public static int log_roll_size_mb = 1024; // 1 GB
 
     /*
-     * Verbose modules. VERBOSE level is implemented by log4j DEBUG level.
-     * eg:
-     *      sys_log_verbose_modules = org.apache.doris.catalog
-     *  This will only print verbose log of files in package org.apache.doris.catalog and all its sub packages.
+     * sys_log_dir:
+     *      This specifies FE log dir. FE will produces 2 log files:
+     *      fe.log:      all logs of FE process.
+     *      fe.warn.log  all WARNING and ERROR log of FE process.
+     *      
+     * sys_log_level:
+     *      INFO, WARNING, ERROR, FATAL
+     *      
+     * sys_log_roll_num:
+     *      Maximal FE log files to be kept.
+     *      
+     * sys_log_verbose_modules:
+     *      Verbose modules. VERBOSE level is implemented by log4j DEBUG level.
+     *      eg:
+     *          sys_log_verbose_modules = org.apache.doris.catalog
+     *      This will only print debug log of files in package org.apache.doris.catalog and all its sub packages.
+     *      
+     * sys_log_roll_interval:
+     *      DAY:  log suffix is yyyyMMdd
+     *      HOUR: log suffix is yyyyMMddHH
      */
+    @ConfField public static String sys_log_dir = System.getenv("DORIS_HOME") + "/log";
+    @ConfField public static String sys_log_level = "INFO"; 
+    @ConfField public static int sys_log_roll_num = 10;
     @ConfField public static String[] sys_log_verbose_modules = {};
+    @ConfField public static String sys_log_roll_interval = "DAY";
```

Comment:
Besides "DAY", what can we choose? can you write them in comment?

Reply:
It IS in comment, see above